ideascube/tests/test_settings.py

```diff
@@ -8,7 +8,7 @@ def test_all_settings(request):
     assert confdir.check(dir=True)
 
     for path in confdir.listdir():
-        if path.ext != 'py':
+        if path.ext != '.py':
```

Comment:
Should we ensure that at least 3 (base, idb and kb) config files were tested?

Maybe keep a counter of how many files have been tested, and assert it is bigger than 3?